CMakeLists.txt

```diff
@@ -11,6 +11,18 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
 # Setup the project
 project(pyne)
 
+
+# Set PYNE version
+set(PYNE_MAJOR_VERSION 0)
+set(PYNE_MINOR_VERSION 7)
+set(PYNE_PATCH_VERSION 5)
```

Comment:
It looks like the version before this PR was `0.7.1`. It seems to me we should keep it the same for now and do a release with this update later. Is that possible?

Reply:
the last version tagged on Github is 0.7.5 that's why I set it like that

Reply:
Oh yeah... that's right. We have advanced release without advancing the version number, which is one of the motivations for this PR